tools/underwater-gps.py

```diff
@@ -6,17 +6,59 @@
 import argparse
 import grequests
 from pymavlink import mavutil
-
+from datetime import datetime
 from os import system
+import operator
+
+# Nmea messages templates
+gpgga = "$GPGGA,{hours:02d}{minutes:02d}{seconds:02.4f},{lat:02.0f}{latmin:02.4f},{latdir},{lon:03.0f}{lonmin:02.4},{londir},1,06,1.2,107.7,M,-33.4,M,,0000*67"
+gprmc = "$GPRMC,{hours:02d}{minutes:02d}{seconds:02.4f},A,{lat:02.0f}{latmin:02.4f},{latdir},{lon:03.0f}{lonmin:02.4},{londir},0.15,{orientation:03.2f},020916,,,A*76"
+gpvtg = "$GPVTG,{orientation:03.2f},T,,M,0.15,N,0.3,K,A*0C"
```

Comment:
Can you put a comment that points to any website our pdf that does how this fields are organized ?

Reply:
done